nrrd/reader.py

```diff
@@ -380,7 +382,7 @@ def read_data(header, fh=None, filename=None):
 
         # Byte skip is applied AFTER the decompression. Skip first x bytes of the decompressed data and parse it using
         # NumPy
-        data = np.fromstring(decompressed_data[byte_skip:], dtype)
+        data = np.frombuffer(decompressed_data[byte_skip:], dtype)           
```

Comment:
Why was this changed to fromstring to frombuffer?

Remove whitespace at end of line.

Also, shouldn't this have an if/else statement for byte_skip == -1 to do something like the following if byte_skip == -1...
```
decompressed_data[-dtype.itemsize * total_data_points:]
```

Reply:
I have already taken care of this condition in the `if` above.

Reply:
The change was a mistake that resulted in failures:
when `np.frombuffer` is used:
 `data.flag['WRITEABLE'] = False`

But when `np.fromstring` is used:
 `data.flag['WRITEABLE'] = True`

The latter is desired in tests.

Reply:
Totally missed that even though I commented on that line...yes, you're right. I like the way you're doing it...

Reply:
Yeah, I think we should make the data writeable so we want to use np.fromstring. I had an issue where I loaded gzip NRRD data and tried to edit it but it didn't work because it wasn't writeable.